aptos-move/e2e-move-tests/src/tests/scripts.rs

```diff
@@ -84,7 +84,7 @@ fn read_coin(h: &MoveHarness, account: &AccountAddress) -> u64 {
 }
 
 #[test]
-fn test_two_to_two_transfer_gas_payer() {
+fn test_two_to_two_transfer_fee_payer() {
```

Comment:
Let's actually move these tests out of scripts.rs into their own file. It has been confusing to find tests as we put too many of them in here even if they're not that strongly related

Reply:
Can we do that in follow-up? I added them here as they are the only one that was using multi agent.